Error in 'Using your own OSM data'

As described `./quickstart.sh my` tries to download `my` from geofabrik

```
====> : Downloading my from Geofabrik...
```

Furthermore, it deletes the file `data/my.osm.pbf` after it exits with an error.

```
root@osm ~/openmaptiles # ls -lh data/my.osm.pbf
-rw-r--r-- 1 root root 51G Apr 30 18:56 data/my.osm.pbf
root@osm ~/openmaptiles # ./quickstart.sh my
=====================================================================================
                                Start processing                                     
-------------------------------------------------------------------------------------
====> : OpenMapTiles quickstart! [ https://github.com/openmaptiles/openmaptiles ]    
      : This will be logged to the ./quickstart.log file (for debugging) and to the screen
      : Area             : my 
      : Git version      : 99f9336e5e493c9f6c7919ee0169d839a9cb0576 
      : Started          : 2020-04-30T19:04+0200 
      : Your bash version: 5.0.3(1)-release
      : Your OS          : linux-gnu
Docker version 19.03.8, build afacb8b7f0
docker-compose version 1.21.0, build unknown
 
-------------------------------------------------------------------------------------
      : This is working on x86_64 ; Your kernel is:
4.19.0-8-amd64
x86_64
      : --- Memory, CPU info ---- 
system memory (GB): 31.3290
SwapTotal:             0 kB
cpu number: 8 x bogomips	: 6800.25
MemFree:          261720 kB
SwapFree:              0 kB
HugePages_Free:        0
 
-------------------------------------------------------------------------------------
====> : Stopping running services & removing old containers
docker-compose down -v --remove-orphans
Removing network openmaptiles_postgres_conn
Removing volume openmaptiles_pgdata
docker-compose rm -fv
No stopped containers
docker volume ls -q -f "name=^openmaptiles_" | xargs --no-run-if-empty docker volume rm
rm -rf cache
 
-------------------------------------------------------------------------------------
====> : Existing OpenMapTiles docker images. Will use version 4.1.0
openmaptiles/import-osm             4.1.0               0ba24517dd5a        3 weeks ago         1.14GB
openmaptiles/openmaptiles-tools     4.1.0               b9e312ddc59d        3 weeks ago         1.27GB
openmaptiles/postgis                4.1.0               4746a366a8d2        3 weeks ago         1.02GB
openmaptiles/import-data            4.1.0               722257ed6444        3 weeks ago         1.28GB
openmaptiles/generate-vectortiles   4.1.0               6984fe80e2de        3 weeks ago         819MB
 
-------------------------------------------------------------------------------------
====> : Create directories if they don't exist
 
-------------------------------------------------------------------------------------
====> : Removing old MBTILES if exists ( ./data/*.mbtiles ) 
 
-------------------------------------------------------------------------------------
====> : Downloading my from Geofabrik...
=============== download-geofabrik =======================
Download area: my
docker-compose run --rm -u 0:0 openmaptiles-tools bash -c \
	'download-osm geofabrik my \
		--minzoom $QUICKSTART_MIN_ZOOM \
		--maxzoom $QUICKSTART_MAX_ZOOM \
		--make-dc /import/docker-compose-config.yml -- -d /import'
Creating network "openmaptiles_postgres_conn" with driver "bridge"
Creating volume "openmaptiles_pgdata" with default driver
Error: ID 'my' was not found in Geofabrik.
Use 'list geofabrik' to see available extract, or try --force-latest to refresh the list of extracts.
make: *** [Makefile:140: download-geofabrik] Error 1

root@osm ~/openmaptiles # ls -lh data/
total 0
```

--- a/QUICKSTART.md
+++ b/QUICKSTART.md
@@ -363,11 +363,11 @@ This is generating `.mbtiles` for your area :  [ MIN_ZOOM: "0"  - MAX_ZOOM: "7" 
 ./quickstart.sh yukon                        # Yukon, Canada
 ```
 ### Using your own OSM data
-Mbtiles can be generated from an arbitrary osm.pbf (e.g. for a region that is not covered by an existing extract) by making the `data/` directory and placing an osm.pbf inside/
+Mbtiles can be generated from an arbitrary osm.pbf (e.g. for a region that is not covered by an existing extract) by making the `data/` directory and placing an *-latest.osm.pbf inside. Inside of folder have to be `docker-compose-config.yml` file too, otherwize whole folder `data/` will be deleted and `download-osm` will try to download osm.pbf file from `geofabric`, `osmfr` or `bbbike`.
 
 ```
 mkdir -p data
-mv my.osm.pbf data/
+mv my-latest.osm.pbf data/
 ./quickstart.sh my
 ```
 

